content/desktop/install/msi/install-and-configure.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Install and configure
+description: Understand how to use the MSI installer. Also explore additional configuration options.
+keywords: msi, windows, docker desktop, install, deploy, configure
+---
+
+## Install 
+
+{{< tabs >}}
+{{< tab name="Install interactively" >}}
+
+TODO
+
+{{< /tab >}}
+{{< tab name="Install from the command line" >}}
+
+TODO
```

Comment:
I can't remember what we agreed for this tab.. it slightly overlaps with installation scenarios iirc.

Would it make sense to move the `Configuration options` section inside the tab?